secondPart/1_videos.js

```diff
@@ -0,0 +1,37 @@
+const convertToJSON = (arr) => {
+    if (Array.isArray(arr)) {
```

Comment:
better to do like this way:

```
if (!Array.isArray) return;

...your code below.
```

This will allow you to avoid code nesting

Reply:
Yes, I like this suggestion. Thanks.